### PR TITLE
Add patched ZK files

### PR DESCRIPTION
**Pull request by jonmv**
@hmusum is this right? :>

### CODE REVIEW

**Review by hmusum — APPROVED** [review]
Looks good to me. Do you log anything that will show that this code is used and not the original one?

**Comment by jonmv** [comment]
Yes, changed the startup logging a bit

**Comment by hmusum** [comment]
👍

**Comment by jonmv** [comment]
@hakonhall FYI

File: zookeeper-server/zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java [hunk 1]
```diff
@@ -0,0 +1,2329 @@
...
+        this.initialConfig = initialConfig;
+
+        this.requestPathMetricsCollector = new RequestPathMetricsCollector();
+
+        this.initLargeRequestThrottlingSettings();
+
+        this.authHelper = new AuthenticationHelper();
+
+        LOG.info(
+            "Created patched server with"
```
Comment (jonmv) [inline zookeeper-server/zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java:363]: Tjis is the log line to look for

**Pull request merged**